CMakeLists.txt

```diff
@@ -12,34 +12,39 @@ function(packageProject)
 
   cmake_parse_arguments(
     PROJECT
-    ""
+    "NO_VERSION_SUFFIX"
     "NAME;VERSION;INCLUDE_DIR;INCLUDE_DESTINATION;BINARY_DIR;COMPATIBILITY;VERSION_HEADER;NAMESPACE"
     "DEPENDENCIES"
     ${ARGN}
   )
 
+  if(NOT DEFINED PROJECT_NO_VERSION_SUFFIX)
+    set(VERSION_POSTFIX -${PROJECT_VERSION})
+  endif()
```

Comment:
`VERSION_POSTFIX` sounds like something that could be defined by an outer project. It might be a good idea to handle the `else` case and set it to an empty string.

Reply:
**`_PROJECT_VERSION_SUFFIX`** seems a better name to me now